### DISCUSSION TITLE
Start libreoffice on container spin up

### DISCUSSION THREAD

**Discussion by willkessler** [gotenberg/gotenberg#445] [Q&A]
gotenberg looks really useful so far. To optimize performance on CloudRun, I'd like to have libreoffice running _before_ the first conversion request arrives (because CloudRun may kill a container any time and replace it, so ideally every container is already running at maximum speed). I noticed testing locally that libreoffice isn't running until the first http request. 

Is there a parameter you can pass to invoke libreoffice as a daemon immediately upon container start? Or is there a simple tweak to the Dockerfile one could make to enable that behavior?  Thanks!!

**Comment by gulien** [comment]
Hello @willkessler,

Versions `7.4.3` and `7.5.0` work exactly like that, but it is causing an issue in some environments (see #420).

The current implementation of the LibreOffice Listener is not optimal, but I lack the time ATM to work on it.
